colossus/src/main/scala/colossus/service/ServiceServer.scala

```diff
@@ -248,8 +252,8 @@ with UpstreamEventHandler[ControllerUpstream[Encoding.Server[P]]]
   }
 
   def processMessages() {
```

Comment:
minor issue, mind adding the return type to these functions?  it's a bit hard to follow